gtsam/slam/ReadMe.md

```diff
@@ -0,0 +1,64 @@
+# SLAM Factors
+
+## GenericProjectionFactor (defined in ProjectionFactor.h)
+
+Non-linear factor for a constraint derived from a 2D measurement. 
```

Comment:
I suggest: "for a constraint derived from a 2D measurement" -> "that minimizes the reprojection error with respect to a 2D measurement".